### DISCUSSION TITLE
疑似因为不当配置导致ip被封禁，下面是一些可能原因

### DISCUSSION THREAD

**Discussion by umisaka** [apernet/hysteria#1250] [General]
1. 使用非443的高位端口，并且未开启端口跳跃
2. 客户端 sni 与服务器伪装域名设置不一致（纯粹自己傻了，瞎改动却忘记编辑客户端配置）
3. 大流量下载行为，大概百十个G的模型文件（可能主要不是这个问题，原因是在改动伪装域名以及端口之前，每月跑到几个T没有出现这个问题）

总之，hy2稳定使用好几个月没什么问题（除了不间断qos），这是被封的第一台。估计是因为前两个误操作导致的，但具体不确定哪个是决定性因素，或许两个一起才是，写出来供大家参考避坑。

**Comment by ghost** [comment]
至于您提到的第 1 点其实并不是最重要的，因为其他使用者也有使用其他端口号的情形，并且也并没有报告什么问题。
您提到的第 2 点比较重要，因为当SNI没有填写且server的地址为域名时，会直接使用server地址来当SNI，所以以后需要注意一下。
第 3 点个人认为并不是直接导致被封锁的原因，因为其他使用者也有类似用途的，但没有导致封锁。

除此之外我自己通过测试还注意到，当server上的hysteria配置档中的quic字段的 maxIdleTimeout: 小于 30s也会导致server第二天被封。

**Comment by ghost** [comment]
换个IP地址重新开始吧，如果方便的话，建议在server本身或者服务提供商的防火墙中设定白名单，只允许需要的端口开放。
